### PR TITLE
ctlv3: cherry pick of #9540 to release 3.3

### PR DESCRIPTION
**Pull request by jingyih**
cc @gyuho

### CODE REVIEW

File: etcdctl/ctlv3/command/ep_command.go [hunk 1]
```diff
@@ -95,15 +103,15 @@ func epHealthCommandFunc(cmd *cobra.Command, args []string) {
...
-	errc := make(chan error, len(cfgs))
+	hch := make(chan epHealth, len(cfgs))
 	for _, cfg := range cfgs {
 		wg.Add(1)
 		go func(cfg *v3.Config) {
 			defer wg.Done()
 			ep := cfg.Endpoints[0]
 			cli, err := v3.New(*cfg)
 			if err != nil {
-				errc <- fmt.Errorf("%s is unhealthy: failed to connect: %v", ep, err)
```
Comment (jingyih) [inline etcdctl/ctlv3/command/ep_command.go]: The output message when this error happens is going to be different due to this change.
@gyuho

**Comment by jingyih** [comment]
This backport introduces changes in output message when fails to connect to an endpoint.

File: etcdctl/ctlv3/command/ep_command.go (hunk 1 above)
Comment (gyuho) [reply etcdctl/ctlv3/command/ep_command.go]: We can highlight this in CHANGELOG-3.3. Could you create another PR for this? Thanks!

File: etcdctl/ctlv3/command/printer_simple.go [hunk 2]
```diff
@@ -141,6 +142,16 @@ func (s *simplePrinter) MemberList(resp v3.MemberListResponse) {
 	}
 }
 
+func (s *simplePrinter) EndpointHealth(hs []epHealth) {
+	for _, h := range hs {
+		if h.Error == "" {
+			fmt.Fprintf(os.Stderr, "%s is healthy: successfully committed proposal: took = %v\n", h.Ep, h.Took)
+		} else {
+			fmt.Fprintf(os.Stderr, "%s is unhealthy: failed to commit proposal: %v", h.Ep, h.Error)
```
Comment (jingyih) [inline etcdctl/ctlv3/command/printer_simple.go]: @gyuho 
When I am trying the failed cases, I noticed there might be an "\n" missing at the end of this message. Can you confirm? If so, I will also need to fix this in master branch and backport the fix.

Comment (jingyih) [reply etcdctl/ctlv3/command/printer_simple.go]: Originally the error messages were sent to `errc` and then printed in separate lines. Here, all errors are printed in one line.

Comment (gyuho) [reply etcdctl/ctlv3/command/printer_simple.go]: Yes, good catch. Can we fix?

Comment (jingyih) [reply etcdctl/ctlv3/command/printer_simple.go]: I'll send out the fix and changelog for these cherry picks later today.

**Pull request merged**